### PR TITLE
fix(core): problem in runtime with bundle itowns and debug

### PR DESCRIPTION
**Pull request by gchoqueux**

### CODE REVIEW

**Comment by gchoqueux** [comment]
I remove useless change in `index.html`

**Review by zarov — APPROVED** [review]
Seems to solve the bbox problem

File: webpack.config.js [hunk 1]
```diff
@@ -1,7 +1,12 @@
 var fs = require('fs');
 var path = require('path');
 
+var webpack = require('webpack');
+
 const debugBuild = process.env.NODE_ENV === 'development';
+var definePlugin = new webpack.DefinePlugin({
+    __DEBUG__: debugBuild,
```
Comment (tbroyer) [inline webpack.config.js]: Isn't this already done in the minify-replace Babel plugin below? Should it replace Babel's minify-replace?

Comment (gchoqueux) [reply webpack.config.js]: yes, thanks, it's a duplicate, I remove it

**Pull request merged**